messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/bridge/JMSBridgeHandler.java

```diff
@@ -82,61 +78,77 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
 
         final JMSBridge bridge = JMSBridge.class.cast(bridgeService.getValue());
 
-        if (READ_ATTRIBUTE_OPERATION.equals(operationName)) {
-            readAttributeValidator.validate(operation);
-            final String name = operation.require(NAME).asString();
-            if (STARTED.equals(name)) {
-                context.getResult().set(bridge.isStarted());
-            } else if (PAUSED.getName().equals(name)) {
-                context.getResult().set(bridge.isPaused());
-            } else {
-                throw MessagingLogger.ROOT_LOGGER.unsupportedAttribute(name);
-            }
-        }
-        else if (START.equals(operationName)) {
-            try {
-                // we do not start the bridge directly but call startBridge() instead
-                // to ensure the class loader will be able to load any external resources
-                JMSBridgeService service = (JMSBridgeService) bridgeService.getService();
-                service.startBridge();
-            } catch (Exception e) {
-                context.getFailureDescription().set(e.getLocalizedMessage());
-            }
-        } else if (STOP.equals(operationName)) {
-            try {
-                bridge.stop();
-            } catch (Exception e) {
-                context.getFailureDescription().set(e.getLocalizedMessage());
-            }
-        } else if (PAUSE.equals(operationName)) {
-            try {
-                bridge.pause();
-            } catch (Exception e) {
-                context.getFailureDescription().set(e.getLocalizedMessage());
-            }
-        } else if (RESUME.equals(operationName)) {
-            try {
-                bridge.resume();
-            } catch (Exception e) {
-                context.getFailureDescription().set(e.getLocalizedMessage());
-            }
-        } else {
+        if (null == operationName) {
             throw MessagingLogger.ROOT_LOGGER.unsupportedOperation(operationName);
```

Comment:
Can 'operationName` be `null`? 
If that's the case, this message might be changed:

> Support for operation null was not properly implemented

is not helpful

Reply:
correct, i should not trust autorefactoing

Reply:
operation name can't be null since it is required